python/cudf/cudf/io/orc.py

```diff
@@ -276,12 +316,14 @@ def read_orc_stripe(orc_file, stripe, columns):
             return pa_table
 
         warnings.warn("Using CPU via PyArrow to read ORC dataset.")
-        orc_file = orc.ORCFile(filepath_or_buffer)
+        orc_file = orc.ORCFile(filepath_or_buffer[0])
```

Comment:
Should we error loudly that we only support single orc file reading via CPU if len(filepath_or_buffer)> 1?

Reply:
You are right we should do something. If a list larger than `1` is provided should we throw a warning or a hard error and just prevent execution if reading via cpu you think? Doesn't really matter to me.

Reply:
Maybe a `NotImplementedError`?

Reply:
> Maybe a `NotImplementedError`?

Yeah, I think a `NotImplementedError` should be fine for now.